### PR TITLE
Updated panda description

### PR DESCRIPTION
**Pull request by cinephilia-z**
This PR updates the panda description & moveit config. Before this, the collision geometry is a cylinder/sphere encasing the hand. The collision area might be too big and the panda hand could not get close to certain object (for example, when we want the robot to grasp a can). Most of the files are copied from moveit_resources package. 

https://github.com/ros-planning/moveit_resources/tree/master/panda_description
https://github.com/ros-planning/moveit_resources/tree/master/panda_moveit_config

### CODE REVIEW

**Comment by zkingston** [comment]
Please double check which files are still needed in the description (the xacro URDF/SRDF files in particular). Remove them if they are no longer needed. Also, please open a pull to update [resources_visualization.cpp](https://github.com/KavrakiLab/robowflex/blob/master/robowflex_library/scripts/resources_visualization.cpp) in the main Robowflex library so the script properly loads the Panda. Also double check which geometry files (the new STLs + the modifications to the existing DAE) are required.

**Comment by cinephilia-z** [comment]
Thx for the reminder! I just deleted the unused xacro urdf/srdf and also updated the panda.yml file.

File: panda/config/cartesian_limits.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cartesian_limits:
```
Comment (ChamzasKonstantinos) [inline panda/config/cartesian_limits.yaml:1]: Delete file

File: panda/config/fake_controllers.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+controller_list:
```
Comment (ChamzasKonstantinos) [inline panda/config/fake_controllers.yaml:1]: Delete File

File: panda/config/panda_controllers.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+ controller_list:
```
Comment (ChamzasKonstantinos) [inline panda/config/panda_controllers.yaml:1]: Delete File

File: panda/config/panda_gripper_controllers.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+ controller_list:
+  - name: position_joint_trajectory_controller
```
Comment (ChamzasKonstantinos) [inline panda/config/panda_gripper_controllers.yaml:2]: Delete File

File: panda/config/sensors_kinect_depthmap.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+sensors:
+  - sensor_plugin: occupancy_map_monitor/DepthImageOctomapUpdater
+    image_topic: /camera/depth_registered/image_raw
```
Comment (ChamzasKonstantinos) [inline panda/config/sensors_kinect_depthmap.yaml:3]: Delete File

File: panda/config/sensors_kinect_pointcloud.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+sensors:
```
Comment (ChamzasKonstantinos) [inline panda/config/sensors_kinect_pointcloud.yaml:1]: Delete File

**Review by ChamzasKonstantinos — CHANGES_REQUESTED** [review]
I marked some other files that you need to delete as well since they are not needed

File: panda.yml [hunk 7]
```diff
@@ -1,5 +1,5 @@
 # Franka Emika Panda robot with hand
-urdf: "package://robowflex_resources/panda/urdf/panda_arm_hand.urdf.xacro"
-srdf: "package://robowflex_resources/panda/config/panda_arm_hand.srdf.xacro"
-joint_limits: "package://robowflex_resources/panda/config/joint_limits.yaml"
+urdf: "package://robowflex_resources/panda/panda_description/urdf/panda.urdf"
+srdf: "package://robowflex_resources/panda/config/panda.srdf"
 kinematics: "package://robowflex_resources/panda/config/kinematics.yaml"
+limits: "package://robowflex_resources/panda/config/joint_limits.yaml"
```
Comment (ChamzasKonstantinos) [inline panda.yml:5]: This should be ``joint_limits``

File: panda.yml [hunk 8]
```diff
@@ -1,5 +1,5 @@
 # Franka Emika Panda robot with hand
-urdf: "package://robowflex_resources/panda/urdf/panda_arm_hand.urdf.xacro"
-srdf: "package://robowflex_resources/panda/config/panda_arm_hand.srdf.xacro"
+urdf: "package://robowflex_resources/panda/panda_description/urdf/panda.urdf"
```
Comment (zkingston) [inline panda.yml:2]: This should follow the prior naming scheme where the urdf is in the "urdf" directory, not the "panda_description" directory.

**Comment by cinephilia-z** [comment]
this is screenshots of running `rosrun robowflex_library resources_visualization panda`
panda_rviz
Screenshot 2021-05-24 181857

**Review by ChamzasKonstantinos — APPROVED** [review]
LGTM!

**Pull request merged**